app/views/conferences/_tickets.html.haml

```diff
@@ -0,0 +1,26 @@
+.container
```

Comment:
This file was renamed to _tickets.haml, so we should delete this file and apply any changes, wherever needed, to the right file.
https://github.com/openSUSE/osem/blob/master/app/views/conferences/_tickets.haml